server/controllers/userAuthController.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable indent */
+import moment from 'moment';
+import uuid from 'uuid';
+import 'babel-polyfill';
```

Comment:
I can see that you used `babel-polyfill`. Can I see know you are doing that? Also, since you have imported it in your index.js file, it would be available to other parts of the app. You do not need more than one import as done here